src/game.js

```diff
@@ -0,0 +1,55 @@
+import { renderRaceResult, renderWinner } from './render.js';
+
+function generateRandomNumber(min, max) {
+  return Math.floor(Math.random() * max) + min;
+}
+
+function playDice() {
+  const randonNumber = generateRandomNumber(1, 9);
+
+  if (randonNumber >= 4) {
+    return true;
+  }
+  return false;
+}
+
+function checkDiceWinner(carObjectArray) {
+  carObjectArray.forEach((car) => {
+    if (playDice()) {
+      car.moveForward();
+    }
+  });
+}
+
+function getMaxPosition(carObjectArray) {
+  let position = 0;
+
+  carObjectArray.forEach((car) => {
+    if (car.position > position) {
+      position = car.position;
+    }
+  });
+  return position;
```

Comment:
```suggestion

  return position;
```

토스트 컨벤션에는 return은 반환하기전에 한칸 공백을 두는게 좋다고 나와있어서 저는 한칸씩 띄어쓰고 있습니다! 아래 링크 첨부하겠습니댜

- https://ui.toast.com/fe-guide/ko_CODING-CONVENTION#%EB%B0%98%ED%99%98%ED%95%98%EA%B8%B0

Reply:
맞아요. 그게 보기 더 좋은 것 같습니다.